src/components/challengeWidget/challengeWidget.js

```diff
@@ -0,0 +1,40 @@
+//import logo from './logo.svg';
+import React from 'react';
```

Comment:
Maybe we should add this component to the `Challenges` skeleton component for showcasing:
https://github.com/bence-toth/challengify/blob/master/src/app/app.js#L94

Reply:
Wow, nicely done! 👏